merlion/models/forecast/ets.py

```diff
@@ -219,8 +245,6 @@ def _forecast(
                 "Trained ETS model is producing NaN forecast. Use the last training point as the prediction."
             )
             forecast[np.isnan(forecast)] = self._last_val
-        if any(np.isnan(err)):
-            err[np.isnan(err)] = 0
```

Comment:
Why remove this? NaN errors can cause issues in downstream code. If it's an issue to return them as 0, maybe you can (1) return 0 if the entire array is NaN, and (2) do linear interpolation otherwise?

Reply:
I see. I have inverted this part